File: Online Judges/Huxley/calculajuroscompostos.c
#include <stdio.h>

int main()
{
  int meses; long double taxa, valor, total;
  scanf("%d\n%LF\n%LF", &meses, &taxa, &valor);
  total = valor;
  while (meses > 0)
  {
    total *= (taxa + 1);
    meses --;
  }
  printf("%.2LF\n", total);
  return(0);
}

File: Online Judges/Huxley/calculajuroscompostos.java
import static java.lang.Math.pow;
import java.text.NumberFormat;
import java.util.Scanner;

public class HuxleyCode
{
  public static void main(String[] args)
  {
    Scanner entrada = new Scanner(System.in);

    int meses = entrada.nextInt();
    double taxa = entrada.nextDouble();
    double valor = entrada.nextDouble();
    double respota = valor * pow((1 + taxa), meses);

    entrada.close();

    NumberFormat nf = NumberFormat.getNumberInstance();
    nf.setMaximumFractionDigits(2);
    nf.setMinimumFractionDigits(2);
    System.out.println(nf.format(respota));
  }
}
